src/pages/Admin/Admin.tsx

```diff
@@ -129,11 +129,9 @@ const VenueDetails: React.FC<VenueDetailsProps> = ({ venueId, roomIndex }) => {
   const { url: matchUrl } = useRouteMatch();
   const { pathname: urlPath } = useLocation();
 
-  const venueSelector = useCallback(
-    (state) => makeVenueSelector(venueId)(state),
-    [venueId]
-  );
-  const venue = useSelector(venueSelector, shallowEqual);
```

Comment:
[note] I forgot that this 'custom comparator'/'shallowEqual' form of `useSelector` existed... I wonder if we can make use of that as part of a 'quickfix' hack to `worldUsers` / etc?